releasenotes/notes/fix-attribute-error-in-flask-2.2.0-94524c33b005d48a.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Flask: version 2.2.0 removes `try_trigger_before_first_request_functions`
```

```suggestion
    Flask: version 2.2.0 removes ``try_trigger_before_first_request_functions``
```


Alternative way we could write it. or maybe this is a "feature" and not a "fix" written this way?

```suggestion
    Flask: fix support for version >= 2.2.0
```

---

Well, to be honest the version checking is `== "2.2.0"` in some places so "2.2.1" would not work as it is

---

> Well, to be honest the version checking is `== "2.2.0"` in some places so "2.2.1" would not work as it is

oh, that might be a problem, because it'll still be gone in 2.2.1 and 2.3.0 it seems